engine/runtime/src/test/java/org/enso/interpreter/test/MetaObjectTest.java

```diff
@@ -148,8 +142,13 @@ private void checkAllTypesSatisfy(Check check) throws Exception {
         continue;
       }
       switch (t.getMetaSimpleName()) {
-        // represented as primitive values without meta object
+        // represented as primitive values without meta objects
+        case "Number" -> {}
         case "Decimal" -> {}
```

Comment:
@JaroslavTulach What should we do with these workarounds? Shouldn't we ensure that for all the values, including primitive types, we have `Value.getMetaObject` return non-null?

Reply:
Primitive values cannot have `getMetaObject`. We would have to wrap all such values into `TruffleObject`. That's not what we want.

> What should we do with these workarounds?

It is not a workaround. Just keep it in.

Reply:
Summarized at https://github.com/enso-org/enso/issues/5759#issuecomment-1443733524 - unless we desperately need `42.getMetaObject()` for stdlib guys, let's let it be.